spec/requests/v1/charts/position_spec.rb

```diff
@@ -34,6 +34,24 @@
     patch position_v1_dashboard_section_charts_path(dashboard_section_id: dashboard_section.id), params: params, headers: user.create_new_auth_token
   end
 
+  context 'when a chart is created in a dashboard section in which there are already charts' do
+    let(:new_chart) { create(:chart, dashboard_section: dashboard_section) }
+
+    it 'has a position value greater than any other chart in the section' do
+      expect(new_chart.position).to eq(Chart.where(dashboard_section_id: dashboard_section.id).maximum(:position))
+      expect(Chart.where(dashboard_section_id: dashboard_section.id, position: new_chart.position).count).to eq(1)
+    end
+  end
+
+  context 'when a chart is created in an empty dashboard section' do
+    let(:new_dashboard_section) { create(:dashboard_section) }
+    let(:new_chart) { create(:chart, dashboard_section: new_dashboard_section) }
+
+    it 'has its position value equal to 1' do
+      expect(new_chart.position).to eq(1)
+    end
+  end
```

Comment:
I don't think you test here anything that you added

Reply:
The previous behavior was to always set `position` to `1` while adding a new chart. The first test checks if it's no longer the case and instead the new `position` is unique and the maximum of all positions in the section the chart belongs to.

The second test would also work before the change, but due to me changing the default behavior (instead of hard-coded `1` now it makes a query and defaults to `1` if no other chart in a section is found) I found it helpful to include a test for this as well